test/integration_test/kubevirt_test.go

```diff
@@ -5,22 +5,30 @@ package integrationtest
 
 import (
 	"fmt"
+	"sync"
 	"testing"
 	"time"
 
 	"github.com/portworx/sched-ops/k8s/core"
 	"github.com/portworx/sched-ops/k8s/kubevirt"
+	"github.com/portworx/sched-ops/k8s/rbac"
 	"github.com/portworx/sched-ops/task"
 	"github.com/portworx/torpedo/drivers/scheduler"
 	"github.com/portworx/torpedo/pkg/log"
 	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/require"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var templatePVCSpecs = map[string]string{
 	"fedora": "kubevirt-templates",
 }
 
+var kubevirtScale = migrationScaleCount
```

Comment:
I don't understand why this is set to migrationScaleCount. Can you please add a comment?

Reply:
This was an error, fixed it.